src/Dapr.Actors/Common/JsonSerializerDefaults.cs

```diff
@@ -0,0 +1,19 @@
+using System.Text.Json;
+
+namespace Dapr.Actors.Common
```

Comment:
Let's not introduce a new namespace, it's just clutter. If its common code it's fine to set in `Dapr.Actors`, and then we don't have extra usings all over.